### PR TITLE
Feature/submit button functionality

### LINKED ISSUE #23: Post fetch data
As a user I would like to add data to the app for steps, hydration, and sleep quality.

### PR DESCRIPTION
**Pull request by mdflynn**
## What is the change?
- adds api request file to contain fetch data
- add submit button and styling to user input field
- add functions to submit and POST user data

## What does it fix?
No fix this PR.

## Is this a fix or a feature?
This is a feature.

## Where should the reviewer start?
- entire api-request file
- lines 219-280 in the styles.css file
- line 61-177 in the scripts.js file

## How should this be tested?
run ```npm start``` in your terminal, then navigate to localhost:8080 to see our new functionality and button

### CODE REVIEW

File: src/api-request.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+let apiRequest = {
```
Comment (cameronRomo) [inline src/api-request.js:1]: Will need to refactor this to have no properties and just have methods.

File: src/css/style.scss [hunk 2]
```diff
@@ -202,3 +216,65 @@ body {
 .user__labels {
   margin: 0 15px 5px 15px;
 }
+
```
Comment (okalimbet) [inline src/css/style.scss:219]: nice use of calc functions to create this amazing rainbow button!

File: src/css/style.scss [hunk 3]
```diff
@@ -5,6 +5,20 @@
   margin: 0;
 }
 
+
+// html,
```
Comment (cameronRomo) [inline src/css/style.scss:9]: This makes for a very nice button the user can enjoy while using the site.

File: src/index.html [hunk 4]
```diff
@@ -20,21 +20,23 @@ <h1 id="headerText">Activity Tracker</h1>
     <form class="user__inputs hidden" id="user__activity__input">
       <label class="user__labels" id="userSteps" for="userStepsInput">Number of Steps:</label><br>
       <input aria-labelledby="userSteps" type="number" min="0" id="number__of__steps" name="userStepsInput" placeholder="Enter Steps.."><br>
-      <label class="user__labels" id="userActive" for="userStepsInput">Active Minutes:</label><br>
-      <input aria-labelledby="userActive" type="number" min="0" id="active__minutes" name="userStepsInput" placeholder="Enter Minutes..">
-      <label class="user__labels" id="userStairs" for="userStepsInput">Flight of Stairs:</label><br>
-      <input aria-labelledby="userStairs" type="number" min="0" id="flights_stairs" name="userStepsInput" placeholder="Enter Flights..">
-      <input type="submit" for="userStepsInput" value="Submit">
+      <label class="user__labels" id="userActive" for="userMinutesInput">Active Minutes:</label><br>
```
Comment (okalimbet) [inline src/index.html:24]: good catch with naming these labels and inputs

File: src/scripts.js [hunk 5]
```diff
@@ -56,28 +58,123 @@ Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydr
...
+
+function getTodaysDate() {
+  var today = new Date();
+  var dd = String(today.getDate()).padStart(2, '0');
+  var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+  var yyyy = today.getFullYear();
+  return today = yyyy + '/' + mm + '/' + dd;
+}
+
+function recordUserActityData() {
```
Comment (cameronRomo) [inline src/scripts.js:114]: Nice to see the 'POST' up and working!

File: src/scripts.js [hunk 6]
```diff
@@ -56,28 +58,123 @@ Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydr
...
   //today = 2019/09/22
   let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData);
   displayRandomUserHistory(randomHistory);
-  generateInitialInfo(userNowId, activityRepo, userRepo, today, randomHistory, userNow);
-  generateActivityInfo(userNowId, activityRepo, today, userRepo, randomHistory, userNow);
+  generateInitialInfo(userNowId, activityRepo, userRepo, today, randomHistory, currentUser);
+  generateActivityInfo(userNowId, activityRepo, today, userRepo, randomHistory, currentUser);
+}
+
+function updateUserInformation() {
```
Comment (okalimbet) [inline src/scripts.js:89]: this is a good place to use switch statement, it fits perfectly. We can consider adjusting default value in the future.

File: src/scripts.js [hunk 7]
```diff
@@ -56,28 +58,123 @@ Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydr
...
+function getTodaysDate() {
+  var today = new Date();
+  var dd = String(today.getDate()).padStart(2, '0');
+  var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+  var yyyy = today.getFullYear();
+  return today = yyyy + '/' + mm + '/' + dd;
+}
+
+function recordUserActityData() {
+  fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData', {
```
Comment (okalimbet) [inline src/scripts.js:115]: love the organization of post requests  as helper functions. Good SRP

**Review by okalimbet — APPROVED** [review]

**Review by cameronRomo — APPROVED** [review]
Overall a very nice feature for posting data!

**Pull request merged**